Exercise_D.js

```diff
@@ -6,15 +6,32 @@
 /*  ====================== */
 /** Write a function that expects a number as a parameter. 
  * If the number that is entered is less than 0, equal to 0, or not a number, 
- * the function should return the string ‘ERROR’. If the number that is entered 
- * is greater than or equal to 1000000 it should simply return the number. 
- * Otherwise it should multiply the number by 10 however many times it takes 
- * to get a number that is greater than or equal to 1000000 and return that number. 
+ * the function should return the string ‘ERROR’. 
+ * 
+ * If the number that is entered is greater than or equal to 1000000 it should simply return the number. 
+ * Otherwise it should multiply the number by 10 however many times it takes to get a number that is greater than or equal to 1000000 and return that number. 
+ 
  * Consider which loop you will need to use to do this.
+ 
+
+
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
-}
+  if (number <= 0 || number == null){
+    return "ERROR";
+  } else if (number >= 1000000){
+    return number;
+  } else {
+    // Multiply number by 10 until it reaches 1000000 or more, then return that number
+    while (number <= 1000000){
+        number *= 10;
+      }
+      return number;
+    }
+  }
+
 
 console.log(multiplyNumber(10));
+

```

Comment:
Superb logic here, great use of pseudo code.

I know you were struggling on the next exercises, so if you come back now and attempt them again I think you will be able to handle it!

Let me know if you want to have a look at it together.